resources/fetching/fetchers/aws/elb_fetcher.go

```diff
@@ -119,4 +119,9 @@ func (r ElbResource) GetMetadata() (fetching.ResourceMetadata, error) {
 		Name:    *r.lb.LoadBalancerName,
 	}, nil
 }
-func (r ElbResource) GetElasticCommonData() (map[string]interface{}, error) { return nil, nil }
+
+func (res ElbResource) GetElasticCommonData() (map[string]interface{}, error) {
+	return map[string]interface{}{
+		"cloud.service.name": "AWS ELB",
```

Comment:
In the ECS example, the cloud provider is not included in the service name:

> Examples: app engine, app service, cloud run, fargate, lambda.

https://www.elastic.co/guide/en/ecs/8.9/ecs-cloud.html#field-cloud-service-name